extensions/Worker.Extensions.Http.AspNetCore/src/Worker.Extensions.Http.AspNetCore.csproj

```diff
@@ -8,7 +8,7 @@
     <!--Version information-->
     <VersionPrefix>2.0.0</VersionPrefix>
     <VersionSuffix>-preview4</VersionSuffix>
```

Comment:
Do we want to remove version suffix in this PR or another?

Reply:
I had the same comment here: https://github.com/Azure/azure-functions-dotnet-worker/pull/2854#discussion_r1838553922 :D